lib/jobi/clients/rabbitmq.rb

```diff
@@ -13,20 +13,22 @@ class Rabbitmq
       }.freeze
 
       def initialize(config = Jobi::Config::Rabbitmq.new)
+        @channels = {}
         @connection = Bunny.new(config.to_h)
         @connection.start
       end
 
-      def channel
-        @channel ||= @connection.create_channel
+      def create_channel(queue_name)
+        @channels[queue_name] ||= @connection.create_channel
       end
 
       def default_exchange
         @default_exchange ||= channel.default_exchange
       end
 
       def queue(name:, options: {})
-        default_exchange
```

Comment:
how does this actually work? is the default exchange used by default?

Reply:
default exchange used to bind the queue with the channel. I forget to remove it before.